locale/de.json

```diff
@@ -29,28 +29,28 @@
     },
     "whitelist_messages": {
         "admin_only": {
-            "mode_title": "Dieser Server ist im <strong>Wartungsmodus</strong>",
-            "insufficient_ids": "Du hast keine <code>Discord</code>oder <code>FiveM</code> identifier, mindestens eins von beiden ist erforderlich, um zu überprüfen, ob du ein txAdmin-Administrator bist.",
-            "deny_message": "Deine identifier sind keinem txAdmin Administrator zugewiesen."
+            "mode_title": "Dieser Server befindet sich derzeit im <strong>Wartungsmodus</strong>",
+            "insufficient_ids": "Du hast keine <code>Discord</code>oder <code>FiveM</code> identifier. Mindestens eine von beiden ist erforderlich, um dich als txAdmin-Mitglied zu registrieren.",
```

Comment:
```suggestion
            "insufficient_ids": "Du hast keinen <code>Discord</code>oder <code>FiveM</code> Identifier. Mindestens einer von beiden ist erforderlich, um dich als registrierten txAdmin-Administrator zu erkennen.",
```

Reply:
that has nothing to do with language mistake. You just rewrote it slightly. It was already correct before.

"Du hast **keine** ID" oder "Du hast **keinen** ID" ?
Korrekt. "Du hast keine ID/Identifier" ist in dem Fall bereits richtig gewesen.

Man ist ebenfalls nicht immer ein txadmin-Administrator, sondern Mitglied mit verschiedenen Permissions. Daher ist das schlichtweg falsch und nicht korrekt.

Reply:
@ProxyPing - Nur weil etwas vorher falsch übersetzt wurde, muss es nicht immer und immer wieder mitgezogen werden. Irgendwann kann man es auch anfangen abzuändern. Niemand außer der Root selbst ist Administrator. Alles andere sind Mitglieder. Aber ja, in der Vergangenheit wurde vieles falsch übersetzt. Ich schau mal auch drüber, danke.

Reply:
Hallo ihr Beiden, ich habe mich dafür an der Originalen Nachricht orientiert: https://github.com/tabarra/txAdmin/blob/master/locale/en.json#L33